scheduler/src/cook/kubernetes/api.clj

```diff
@@ -503,13 +503,21 @@
   (double (+ (bigdec a) (bigdec b))))
 
 (defn adjust-job-resources
-  "Given the required resources for a job, add to them the resources for any sidecars that will be launched with the job."
-  [{:keys [cpus mem] :as resources}]
-  (if-let [{:keys [cpu-request memory-request]} (-> (config/kubernetes) :sidecar :resource-requirements)]
-    (assoc resources
-      :cpus (add-as-decimals cpus cpu-request)
-      :mem (add-as-decimals mem memory-request))
-    resources))
+  "Adjust required resources for a job, adding any additional resources. Additional resources might be needed for
+   sidecars or extra processes that will be launched with the job."
```

Comment:
I think we should add a line to the docstring about how/where this function gets used.